Servlets/src/main/java/ua/goit/servlets/servlets/CategoriesServlet.java

```diff
@@ -0,0 +1,34 @@
+package ua.goit.servlets.servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import ua.goit.servlets.dao.CategoriesList;
+import ua.goit.servlets.model.Category;
+
+public class CategoriesServlet extends HttpServlet {
+  static final long serialVersionUID = 1590679L;
+  List<Category> categories; //visible for testing
+
+  public void init() throws ServletException {
+	super.init();
```

Comment:
indentation should be consistent across the file. `Ctrl + Shift+ I` in eclipse to fix it automatically